scheduler/src/cook/mesos/task.clj

```diff
@@ -17,7 +17,7 @@
   (:require [clojure.data.json :as json]
             [clojure.set :as set]
             [clojure.tools.logging :as log]
-            [cook.config :refer (config)]
+            [cook.config :refer (config, executor-config)]
```

Comment:
Nitpick: I have a preference for qualifying calls to the method with the alias, e.g. `[cook.config :as config]` and `(config/executor-config)`

Reply:
Done